integrations/whatsapp/src/index.ts

```diff
@@ -198,7 +203,8 @@ const integration = new bp.Integration({
         const signatureHash = signature.split('=')[1]
         const expectedHash = crypto.createHmac('sha256', secret).update(req.body).digest('hex')
         if (signatureHash !== expectedHash) {
-          const errorMessage = "Couldn't validate the request signature."
+          const errorMessage =
+            "Couldn't validate the request signature, please verify the client secret configuration property!"
```

Comment:
nit: exclamation marks at the end of an error can sound a bit harsh 😇 

```suggestion
            "Couldn't validate the request signature, please verify the client secret configuration property."
```